js/atoms/src/Block.svelte

```diff
@@ -35,6 +35,7 @@
 	style:overflow={allow_overflow ? "visible" : "hidden"}
 	style:flex-grow={scale}
 	style:min-width={`calc(min(${min_width}px, 100%))`}
+	style:border-width="var(--block-border-width)"
```

Comment:
no idea why I needed to do this but was not picking up the css variable otherwise